Discord-UWP/SharedModels/TypingStart.cs

```diff
@@ -0,0 +1,19 @@
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Discord_UWP.SharedModels
+{
+    public class TypingStart
```

Comment:
Need to add this class to https://github.com/gantonious/Discord-UWP/blob/master/Discord-UWP/Discord-UWP.csproj#L176 so it will compile in.